[Joy] Add template UIs & first look blog post (#32791)
mui/material-ui: docs/data/joy/components/aspect-ratio/aspect-ratio.md

<p class="description">The Aspect Ratio component shapes the content with the specified ratio.</p>

> The default Aspect Ration implementation combines `height: 0px` with percentage padding-bottom to create the proper aspect ratio of the content.

Reviewer: ```suggestion
The default aspect ratio implementation combines `height: 0px` with percentage padding-bottom to create the proper aspect ratio for the content.
```

> (It will be replaced by the [CSS aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) in the future once it has sufficient various browser support).

Reviewer: ```suggestion
(It will be replaced by the [CSS aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) in the future once it has sufficient browser support).
```

- The content to fit the area of the aspect ratio component must be its first direct child.

> - It is designed to be composed with other components, e.g. [`Card`](/joy-ui/react-card/).

Reviewer: ```suggestion
- It's designed to be composed with other components, e.g. [`Card`](/joy-ui/react-card/).
```
